### PR TITLE
Roadmap 2024

### PR DESCRIPTION
**Pull request by ceb8**
Updating the Astropy Roadmap after community input and discussion as the Astropy Coordination Meeting. As a result of the discussion at the Astropy Coordination Meeting each category of the Roadmap has had a brief decription/vision of the category added.

This PR closes https://github.com/astropy/astropy-project/issues/432.

### CODE REVIEW

File: roadmap/roadmap.md [hunk 1]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+- :large_orange_diamond: Revive dashboards for developers (CI status etc) and community (PR trends etc).
+
 
-- :large_orange_diamond: Improve support and validation for input/output of standard file formats used in Astronomy by improving the unified I/O system.
+## Interoperability and compatibility
 
-- :large_orange_diamond: Improve interoperability and compatibility, while reducing duplication, amongst Astropy-affiliated packages and other community standard software.
+One of the core principles of the Astropy Project is to "foster an ecosystem of interoperable astronomy packages." This category encompasses goals related to that principle of interoperability and compatibility.
+
+- :green_circle: Improve support and validation for input/output of standard file formats used in Astronomy by improving the unified I/O system: allowing user added classes/formats, and implementing GWCS.
```
Comment (pllim) [inline roadmap/roadmap.md:65]: Clarify acronym. We don't have to capitalize "astronomy," do we?
```suggestion
- :green_circle: Improve support and validation for input/output (I/O) of standard file formats used in astronomy by improving the unified I/O system: allowing user added classes/formats, and implementing GWCS.
```

File: roadmap/roadmap.md [hunk 2]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+
+- :red_square: Unify unit packages.
```
Comment (pllim) [inline roadmap/roadmap.md:67]: Would listing them out be too specific?
```suggestion
- :red_square: Unify unit packages (e.g., `astropy.units`, `pint`, `unyt`).
```

File: roadmap/roadmap.md [hunk 3]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+    
 
 ## Hardware and Performance
 
+Astronomers perform their work in a variety of computing environments with a variety of requirements. This category encompasses goals related to supporting the performance and hardware needs of Astropy Project users, as well as  anticipating future needs and requirements.
```
Comment (pllim) [inline roadmap/roadmap.md:72]: Exorcise extra whitespace.
```suggestion
Astronomers perform their work in a variety of computing environments with a variety of requirements. This category encompasses goals related to supporting the performance and hardware needs of Astropy Project users, as well as anticipating future needs and requirements.
```

File: roadmap/roadmap.md [hunk 4]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+
 - :large_orange_diamond: Improve and/or maintain interoperability with performant I/O file formats and libraries such as HDF5 and Dask.
 
 - :red_square: Improve support for using Astropy tools in heterogeneous computing environments such as cloud environments or GPU systems.
 
+- :red_square: Add basic pyscript/webassembly support in Astropy.
```
Comment (pllim) [inline roadmap/roadmap.md:78]: JS people love their CamelCase.
```suggestion
- :red_square: Add basic PyScript/WebAssembly support in Astropy.
```

File: roadmap/roadmap.md [hunk 5]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+
+
 ## Learn and User Support
 
+The Astropy Project cannot succeed in its core objectives without a robust user onboarding and support system.  This category is for items related to the Astropy Learn ecosystem and other user support related improvements.
```
Comment (pllim) [inline roadmap/roadmap.md:83]: Can you tell that double whitespace bothers me? 😆 
```suggestion
The Astropy Project cannot succeed in its core objectives without a robust user onboarding and support system. This category is for items related to the Astropy Learn ecosystem and other user support related improvements.
```

File: roadmap/roadmap.md [hunk 6]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+-  :large_orange_diamond: Update Astropy website to improve the visibility of Astropy Learn and overall reflect the Project as a whole, not just the core package.
 
-- :large_orange_diamond: Expand and build on workshop offerings by recruiting and training more facilitators and expanding the geographic diversity of meetings where we offer workshops. 
+- :large_orange_diamond: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
 
-- :large_orange_diamond: Expand the number and diversity of active user support resources.
+- :red_square: Expand and build on workshop offerings, including online and increased geographic coverage.
 
-- :red_square: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
+- :red_square: Develop/harvest tutorials suitable for use in university astronomy courses.
```
Comment (pllim) [inline roadmap/roadmap.md:93]: Do we want it more vague to include non-traditional tertiary education that we are not aware of?
```suggestion
- :red_square: Develop/harvest tutorials suitable for use in post-secondary academic settings.
```

File: roadmap/roadmap.md [hunk 7]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
-- :red_square: Develop/harvest tutorials suitable for use in university astronomy courses. 
 
 ## Community Building and Sustainability
 
-- :large_orange_diamond: Increase the learning and mentoring opportunities for people interested in becoming contributors and helping to develop existing contributors.
+As a commnunity driven project the long term health and development of the Astropy community is an intrinsic part of the Project. This category encompasses goals related to growing and maintining that community, and ensuring it adheres to the core principles of inclusion and consensus-building.
```
Comment (pllim) [inline roadmap/roadmap.md:98]: Minor word change suggestions. Also typos.
```suggestion
As a community-driven project, the long term health and development of the Astropy community is an intrinsic part of the Project. This category encompasses goals related to growing and maintaining that community, and ensuring that it adheres to the core principles of inclusion and consensus-building.
```

File: roadmap/roadmap.md [hunk 8]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
-- :large_orange_diamond: Better understand Astropy user community through a NumPy-like user survey (see [this summary](https://numpy.org/user-survey-2020-details/) of their results).
+- :green_circle: Better understand Astropy user community through a NumPy-like user survey (see [this summary](https://numpy.org/user-survey-2020-details/) of their results).
```
Comment (pllim) [inline roadmap/roadmap.md:100]: Is this referring to the Mycology reports we discussed? Should we link them here?

File: roadmap/roadmap.md [hunk 9]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
-- :red_square: Increase inclusion, diversity, and empowerment efforts within the Astropy Project and NumFOCUS communities, and improve our understanding of the demographics of our communities in order to measure the effectiveness of these efforts.
+- :large_orange_diamond: Increase the learning and mentoring opportunities for people interested in becoming contributors and helping to develop existing contributors.
```
Comment (pllim) [inline roadmap/roadmap.md:102]:
```suggestion
- :large_orange_diamond: Increase the learning and mentoring opportunities for people interested in becoming contributors and helping to develop existing contributors into maintainers.
```

File: roadmap/roadmap.md [hunk 10]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
-## Infrastructure, Documentation
+- :red_square: Increase inclusion, diversity, and empowerment efforts within the Astropy Project and NumFOCUS communities, and improve our understanding of the demographics of our communities in order to measure the effectiveness of these efforts.
 
-- :green_circle: Implement integration testing for core, coordinated, and infrastructure packages.
 
-- :large_orange_diamond: Improve documentation for infrastructure setup, as well as the developer documentation.
+## Government, Management, and Personnel
 
-- :red_square: Implement robust performance benchmark reporting.
+This category encompasses goals related to governing and managing the people of the Astropy Project to facilitate reaching the goals described elsewhere in this document.
```
Comment (pllim) [inline roadmap/roadmap.md:109]:
```suggestion
This category encompasses goals related to governing and managing the teams of the Astropy Project to facilitate reaching the goals described elsewhere in this document.
```

File: roadmap/roadmap.md [hunk 11]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
-## Government, Management, and Personnel
+- :green_circle: Develop a process for recruiting, selecting, and managing personnel to complete Project priorities that require external labor.
 
 - :large_orange_diamond: Adopt the NumFOCUS Code of Conduct (CoC) and build a community ombuds/CoC team to curate the guidelines and enforce them.
 
-- :large_orange_diamond: Develop a process for recruiting, selecting, and managing personnel to complete Project priorities that require external labor.
+- :large_orange_diamond: Document the process for taking on a formal role in the Astropy project and recruit for unfilled Team positions.
```
Comment (pllim) [inline roadmap/roadmap.md:115]: No need to capitalize "team?"
```suggestion
- :large_orange_diamond: Document the process for taking on a formal role in the Astropy project and recruit for unfilled team positions.
```

File: roadmap/roadmap.md [hunk 12]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
-- :red_square: Document the process for taking on a formal role in the Astropy project and recruit for unfilled Team positions.
+- :large_orange_diamond: Implement a strategic planning role.
```
Comment (pllim) [inline roadmap/roadmap.md:117]:
```suggestion
- :large_orange_diamond: Define and staff the strategic planning role.
```

**Review by pllim — COMMENTED** [review]
Thanks for putting this together! Just minor comments.

File: roadmap/roadmap.md (hunk 1 above)
Comment (ceb8) [reply roadmap/roadmap.md:65]: I probably originally wrote Astropy and didn't change the capitalization when I changed the wording.

File: roadmap/roadmap.md (hunk 5 above)
Comment (ceb8) [reply roadmap/roadmap.md:83]: Can you tell I don't even see it. 😉

File: roadmap/roadmap.md (hunk 6 above)
Comment (ceb8) [reply roadmap/roadmap.md:93]: I don't think so. I don't want to over-promise and this is not saying that these tutorials are *only* for university courses, just that we are building a collection that is *suitable* for university courses, and if they are suitable for other courses too, great.

File: roadmap/roadmap.md [hunk 13]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+
+- :large_orange_diamond: Provide next-generation spectroscopic reduction tools usable by individual researchers and larger surveys.
+
+
+## Infrastructure, Documentation
+
+Underlying all Astropy packages is critial infrastucture that ensures high quality, well-documented software is released. This category emcompasses goals having to do with this infrastructure and documentation.
+
+
+- :green_circle: Implement robust performance benchmark reporting.
```
[the remark this answers is not in the document]
Comment (ceb8) [reply roadmap/roadmap.md:54]: I think that is too specific.

File: roadmap/roadmap.md [hunk 14]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+## Infrastructure, Documentation
+
+Underlying all Astropy packages is critical infrastructure that ensures high quality, well-documented software is released. This category encompasses goals having to do with such infrastructure and documentation.
+
+
+- :green_circle: Implement robust performance benchmark reporting.
+
+- :large_orange_diamond: Improve documentation for infrastructure setup, as well as the developer documentation.
+
+- :large_orange_diamond: Revive dashboards for developers (CI status etc) and community (PR trends etc).
```
Comment (jeffjennings) [inline roadmap/roadmap.md:58]: I'm working now on a community dashboard, so that could be green.

File: roadmap/roadmap.md [hunk 15]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+## Infrastructure, Documentation
+
+Underlying all Astropy packages is critical infrastructure that ensures high quality, well-documented software is released. This category encompasses goals having to do with such infrastructure and documentation.
+
+
+- :green_circle: Implement robust performance benchmark reporting.
+
+- :green_circle: Revive dashboards for developers (CI status etc) and community (PR trends etc).
+
+- :large_orange_diamond: Improve documentation for infrastructure setup, as well as the developer documentation.
```
Comment (pllim) [inline roadmap/roadmap.md:58]: Actually, @taldcroft already improved developer docs in https://github.com/astropy/astropy/pull/16561 . Does that mean we can take developer docs off this roadmap? Or change status for that one?

File: roadmap/roadmap.md [hunk 16]
```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
-- :red_square: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
+- :red_square: Develop/harvest tutorials suitable for use in university astronomy courses.
 
-- :red_square: Develop/harvest tutorials suitable for use in university astronomy courses. 
 
 ## Community Building and Sustainability
 
-- :large_orange_diamond: Increase the learning and mentoring opportunities for people interested in becoming contributors and helping to develop existing contributors.
+As a commnunity-driven project, the long term health and development of the Astropy community is an intrinsic part of the Project. This category encompasses goals related to growing and maintaining that community, and ensuring that it adheres to the core principles of inclusion and consensus-building.
```
Comment (pllim) [inline roadmap/roadmap.md:98]:
```suggestion
As a community-driven project, the long term health and development of the Astropy community is an intrinsic part of the Project. This category encompasses goals related to growing and maintaining that community, and ensuring that it adheres to the core principles of inclusion and consensus-building.
```

File: roadmap/roadmap.md (hunk 15 above)
Comment (ceb8) [reply roadmap/roadmap.md:58]: Let's change to green and leave on.

**Comment by ceb8** [comment]
@pllim I think this is ready to merge now.

**Review by pllim — APPROVED** [review]
Thanks!

**Pull request merged**